core/src/main/scala/ai/h2o/sparkling/backend/internal/InternalBackendConf.scala

```diff
@@ -83,20 +83,6 @@ trait InternalBackendConf extends SharedBackendConf {
   }
 
   def setSpreadRddRetriesTimeout(timeout: Int): H2OConf = set(PROP_SPREADRDD_RETRIES_TIMEOUT._1, timeout.toString)
-
-  def internalConfString: String =
```

Comment:
why this was moved to a dedicated trait?

Reply:
We get all the methods defined in the *Conf classes and automatically generated getters & setters. This method is not supposed to be part of the generated configuration

Reply:
I see, thanks